packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -53,6 +53,7 @@ export interface DropdownSlotClassNames {
   selectedItem: string
   selectedItems: string
   triggerButton: string
+  inverted: string
```

Comment:
Don't need to add it as a slot.

Slots are used on complex components. For instance, a checkbox can have a label and a ticking box, so we will add those as slots to the checkbox component. Useful to apply styles.

Here `inverted` is just a prop, unless I am missing something.

Reply:
@silviuavram is right, this is just a regular prop, that should appear in the `DropdownProps` and `propTypes`, not in the `DropdownSlotClassNames`